src/common/m_checker_common.fpp

```diff
@@ -34,10 +36,10 @@ contains
 #endif
 
 #ifndef MFC_POST_PROCESS
-        if (bubbles) call s_check_inputs_bubbles
```

Comment:
how are you able to remove this?

Reply:
They are now within the subroutine, so the error message will contain the bubble condition:

```
    subroutine s_check_inputs_bubbles
        @:PROHIBIT(bubbles .and. nb < 1, 'The Ensemble-Averaged Bubble Model requires nb >= 1')
        @:PROHIBIT(bubbles .and. polydisperse .and. (nb == 1), 'Polydisperse bubble dynamics requires nb > 1')
        @:PROHIBIT(bubbles .and. polydisperse .and. (mod(nb, 2) == 0), 'nb must be odd')
        @:PROHIBIT(bubbles .and. (.not. polytropic) .and. f_is_default(R0ref), 'R0ref must be set if using bubbles with polytropic = .false.')
        @:PROHIBIT(bubbles .and. nb == dflt_int, 'nb must be set if using bubbles')
        @:PROHIBIT(bubbles .and. thermal > 3)
        @:PROHIBIT(bubbles .and. model_eqns == 3, 'Bubble models untested with 6-equation model (model_eqns = 3)')
        @:PROHIBIT(bubbles .and. model_eqns == 1, 'Bubble models untested with pi-gamma model (model_eqns = 1)')
        @:PROHIBIT(bubbles .and. model_eqns == 4 .and. f_is_default(rhoref), 'rhoref must be set if using bubbles with model_eqns = 4')
        @:PROHIBIT(bubbles .and. model_eqns == 4 .and. f_is_default(pref), 'pref must be set if using bubbles with model_eqns = 4')
        @:PROHIBIT(bubbles .and. model_eqns == 4 .and. num_fluids /= 1, '4-equation model (model_eqns = 4) is single-component and requires num_fluids = 1')
        @:PROHIBIT(bubbles .and. cyl_coord, 'Bubble models untested in cylindrical coordinates')
    end subroutine s_check_inputs_bubbles
```

Reply:
Fair enough, but can't you wrap the subroutine in an `if (bubbles) then` statement and eliminate the `(bubbles .and.` part of all of these?

Reply:
It's that I want the error message to be:
`Prohibited condition: bubbles .and. thermal > 3`
instead of just:
`Prohibited condition: thermal > 3`

Reply:
Oh, sorry, yes, this is obvious to me now. Forgive me; I am jetlagged and sending around a lot of nonsense.